src/source/Metrics/Metrics.c

```diff
@@ -9,7 +9,7 @@ STATUS getIceCandidatePairStats(PRtcPeerConnection pRtcPeerConnection, PRtcIceCa
     STATUS retStatus = STATUS_SUCCESS;
     BOOL locked = FALSE;
     PIceAgent pIceAgent = NULL;
-    CHK((pRtcPeerConnection != NULL || pRtcIceCandidatePairStats != NULL), STATUS_NULL_ARG);
+    CHK((pRtcPeerConnection != NULL && pRtcIceCandidatePairStats != NULL), STATUS_NULL_ARG);
```

Comment:
Perhaps we can just change this to if "either are null" rather than "if both are not null". (more readable that way)

Reply:
That was the check. But `CHK` has a !(cond), so using `&&`